### ISSUE TITLE
More fixes required for Mac OS install instructions

### ISSUE THREAD

**Issue by ttimbers** [UBC-MDS/UBC-MDS.github.io-jekyll#62]
@joelostblom - there are a few more small things to fix for the Mac OS install instructions:

- [x] Link to R instructions ("R, XQuartz,IRkernel and RStudio" in TOC) is broken
- [x] There are no instructions provided that Mac users should add psql to their `PATH`, nor how to do this.

Can you please make these changes? Thanks!

**Comment by joelostblom** [comment]
I thought again about the adding of postgres to path for macOS. One thing I don't like is that the version number is included the application path `/Applications/PostgreSQL 12/SQL Shell (psql).app`. I think this can be confusing for students if they update postgres and it suddenly is not available from terminal anymore because the path changed...

Does it work to get the version of postgres the same as for Rstudio? Then I can add that to the special macOS section in the script so that it is up to us to update it when the postgres version changes, but students never have to deal with future updates breaking their PATH changes. What do you think?

```
mdls -name kMDItemVersion /Applications/PostgreSQL 12/SQL Shell (psql).app
```

**Comment by joelostblom** [comment]
Also note some confusion on how to add postgres to path in this SO thread https://stackoverflow.com/questions/20027374/why-cant-i-add-postgresql-to-my-path

**Comment by ttimbers** [comment]
Remember, if we can also get the version on Mac this way:

```
/Library/PostgreSQL/12/bin/psql --version
```

```
psql (PostgreSQL) 12.3
```

**Comment by joelostblom** [comment]
Ah I didn't realize there existed a psql executable that was not the .app shell as for rstudio, great! I have made this update in the script

**Comment by joelostblom** [comment]
Closed by #63 #67
